### PR TITLE
feat(tanstack-table): make column linkable by another column

### PR DESCRIPTION
**Pull request by ahmeterenodaci**
## Screenshots:
resim

## Types of changes

- [ ] Chore (non-breaking change which improve code)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist:

- [ ] Added story.
- [ ] Checked relative components.
- [ ] Checked the component on production.
- [ ] Added related docs.

### CODE REVIEW

File: src/molecules/tanstack-table/utils.tsx [hunk 1]
```diff
@@ -38,20 +39,49 @@ export function getCommonPinningStyles<TData>({
...
+        `/${row
+          ._getAllCellsByColumnId()
+          ?.[link.targetAccessorKey || ''].getValue()
+          ?.toString()}` || '';
+    }
+    if (link.suffix) {
+      url += `/${link.suffix}`;
+    }
+    return (
+      <a href={url} className="font-medium underline">
```
Comment (ertugrulcan-ays) [inline src/molecules/tanstack-table/utils.tsx:65]: can we don't use "a" tag by default I prefer next "Link" or we can pass as prop

File: src/molecules/tanstack-table/tanstack-table.stories.tsx [hunk 2]
```diff
@@ -19,6 +19,16 @@ Default.args = {
...
+    links: {
+      userName: {
+        targetAccessorKey: 'id',
+        prefix: 'http://192.168.1.105:1453/tr/app/admin',
+        suffix: '/edit',
+      },
+      email: {
+        prefix: 'http://192.168.1.105:1453/tr/app/',
+      },
+    },
```
Comment (a0m0rajab) [inline src/molecules/tanstack-table/tanstack-table.stories.tsx:31]: I feel we can have a direct way to change the cells instead of focusing on links only/

**Review by a0m0rajab — APPROVED** [review]

File: src/molecules/tanstack-table/utils.tsx (hunk 1 above)
Comment (a0m0rajab) [reply src/molecules/tanstack-table/utils.tsx:65]: We have next in the project.

Comment (eren-ayasofyazilim) [reply src/molecules/tanstack-table/utils.tsx:65]: I will change this in next pr.

File: src/molecules/tanstack-table/tanstack-table.stories.tsx (hunk 2 above)
Comment (eren-ayasofyazilim) [reply src/molecules/tanstack-table/tanstack-table.stories.tsx:31]: In next pr we will able to show icon. Do you mean something other than this?

**Pull request merged**